### ISSUE TITLE
Fuzzel + clipboard not working (keybinds)

### ISSUE THREAD

**Issue by Deivis44** [end-4/dots-hyprland#805]
- I have read the Usage and Troubleshooting pages of the wiki: https://end-4.github.io/dots-hyprland-wiki/en/i-i/03troubleshooting/
- I have made sure that both my config and system packages are up to date
- Linux distro: Arch

---
## The issue
I was using normally this config and I remember that a new version from fuzzel was released (I think) later as I normally use, the command for watch my clipboard but in this case was not working, so I was checking the keybinds and found those:

```
bind = Super, V, exec, pkill fuzzel || cliphist list | fuzzel --no-fuzzy --dmenu | cliphist decode | wl-copy # Clipboard history >> clipboard
bind = Super, Period, exec, pkill fuzzel || ~/.local/bin/fuzzel-emoji # Pick emoji >> clipboard
```

Trying to modify them and found that both commands are not working for the argument `pkill` in `... exec, pkill fuzzel || ...` you can delete it but now the keybinds don't do their purpouse, idk how to solve it

Also, I literally reboot my computer, install everything again and the problem is the same :(

**Comment by Deivis44** [comment]
Ok, watching at the fuzzel repo, the argument `--no-fuzzy` is not available now

**Comment by l4v3nx** [comment]
change `--no-fuzzy` to `--match-mode fzf` (there are also options "exact" and "fuzzy". check `man fuzzel.ini`)

**Comment by 1677883418** [comment]
> change `--no-fuzzy` to `--match-mode fzf` (there are also options "exact" and "fuzzy". check `man fuzzel.ini`)将 ``--no-fuzzy`` 更改为 ``--match-mode fzf``（还有选项 "exact" 和 "fuzzy"。请参阅 ``man fuzzel.ini``）

resolved my question!Thanks

**Comment by maxz255** [comment]
> change `--no-fuzzy` to `--match-mode fzf` (there are also options "exact" and "fuzzy". check `man fuzzel.ini`)

thx, I have same  issue

**Comment by so-uvik** [comment]
The clipboard functionality works but not the emoji selector, got any solutions?
`bind = Super, Period, exec, pkill fuzzel || ~/.local/bin/fuzzel-emoji # Pick emoji >> clipboard`

**Comment by end-4** [comment]
fixed in #810
